src/services/ui/src/features/package/package-details/appk.tsx

```diff
@@ -12,13 +12,15 @@ import { usePackageDetailsCache } from "..";
 
 export const AppK = () => {
   const [removeChild, setRemoveChild] = useState("");
+  const [loading, setLoading] = useState(false);
   const { data: user } = useGetUser();
   const cache = usePackageDetailsCache();
   const submission = useMutation({
     mutationFn: (config: SubmissionServiceParameters<any>) => submit(config),
   });
```

Comment:
This can actually just be `useSubmissionService` which is exported from `submissionService.ts`